website/static/js/osfHelpers.js

```diff
@@ -224,6 +224,25 @@ var unblock = function() {
     $.unblockUI();
 };
 
+var blockElement = function($el, message) {
+    $el.block({
+        message: message,
+        css: {
+            border: 'none',
+            padding: '15px',
+            backgroundColor: '#000',
+            '-webkit-border-radius': '10px',
+            '-moz-border-radius': '10px',
+            opacity: 0.5,
+            color: '#fff'
+        }
+    });
+};
+
+var unblockElement = function($el) {
```

Comment:
Both `blockElement` and `unblockElement` appear to be unused and untested. I've removed them.